app/assets/javascripts/app/form/remote_requests.js

```diff
@@ -0,0 +1,36 @@
+var RemoteRequestsForm = function (el) {
+  this.$el = el;
+
+  this.onAjaxSuccess = function (e, data) {
+    this.$el.find('.success')
+      .removeClass('w-hidden');
+  };
+
+  this.onAjaxError = function (e, data) {
+    var response = jQuery.parseJSON(data.responseText);
+    var $errorCard = this.$el.find('.card-error');
```

Comment:
Esse `$errorCard` está idêntico em outros lugares, será que não seria melhor colocar ele lá junto com o `this.$el`? ou então criar uma função para ele?